.gitignore

```diff
@@ -1 +1,2 @@
+.idea
```

Comment:
Please ignore this in your computer only. 💻 

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer